test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -2928,13 +2889,128 @@ const char kServiceConfigOob[] =
     "    {\"weighted_round_robin_experimental\": {\n"
     "      \"blackoutPeriod\": \"0s\",\n"
     "      \"weightUpdatePeriod\": \"0.1s\",\n"
+    "      \"weightExpirationPeriod\": \"2s\",\n"
     "      \"enableOobLoadReport\": true\n"
     "    }}\n"
     "  ]\n"
     "}";
 
+class WeightedRoundRobinTest : public ClientLbEnd2endTest {
+ protected:
+  void ExpectWeightedRoundRobinPicks(
+      const grpc_core::DebugLocation& location,
+      const std::unique_ptr<grpc::testing::EchoTestService::Stub>& stub,
+      const std::vector<size_t>& expected_weights, size_t total_passes,
```

Comment:
Suggest making `total_passes` default to 3.

Reply:
Done.